app/src/main/java/io/github/droidkaigi/confsched2018/presentation/search/SearchFragment.kt

```diff
@@ -154,6 +155,12 @@ class SearchFragment : Fragment(), Injectable {
             }
         })
         changeSearchViewTextColor(searchView)
+
+        searchView.setOnQueryTextFocusChangeListener { view, b ->
+            val imm = view.context.getSystemService(Context.INPUT_METHOD_SERVICE) as
```

Comment:
🙋 
`if (b) {` is unnecessary here?

Reply:
@konifar Thank you for comment.
Actually without `if (!b) {`, this code seems working.
I checked stack trace and found that when pushed search button, `setImeVisibility(true)` ran after requesting focus.
In other words, IME is set to invisible then set to visible.
However, I think checking `!b` is better to do since no need to hide keyboard when query text field has focus and timing to set IME visible may change in future.
I'll add `if (!b) {` surround code which hiding IME.